src/hb-ot-cmap-table.hh

```diff
@@ -1538,7 +1594,14 @@ struct cmap
     unsigned format4objidx = 0, format12objidx = 0, format14objidx = 0;
     auto snap = c->snapshot ();
 
-    SubtableUnicodesCache unicodes_cache (base);
+    SubtableUnicodesCache local_unicodes_cache (base);
+    SubtableUnicodesCache* unicodes_cache = &local_unicodes_cache;
+
+    if (plan->accelerator &&
+        plan->accelerator->cmap_cache &&
+        plan->accelerator->cmap_cache->same_base (base))
```

Comment:
Why do you need the `same_base` check here?

Reply:
I can see it as a good idea but curious why you added.

Reply:
Actually, unless you always retain the old blob, this is racy since old and new blob might have the same base pointer address.

Reply:
It looks to me like it should always be true and its a safety check. I''m happy with that. Or maybe change it to an assert.

Reply:
That might fail indeed if the face loader returns new copies of tables each time...

Reply:
This is mostly a safety check in the case of the situation such as where reference table references a different memory address each time it's called. In that case we just avoid using the cache.